libs/vertexai/langchain_google_vertexai/_anthropic_parsers.py

```diff
@@ -26,27 +26,20 @@ def parse_result(self, result: List[Generation], *, partial: bool = False) -> An
         """
         if not result or not isinstance(result[0], ChatGeneration):
             return None if self.first_tool_only else []
+
         message = result[0].message
+        tool_calls: List[Any] = []
+
         if isinstance(message, AIMessage) and message.tool_calls:
             tool_calls = message.tool_calls
-        elif isinstance(message.content, str):
-            tool_calls: List = []
-        else:
-            content: List = message.content
-            _tool_calls = [dict(tc) for tc in _extract_tool_calls(content)]
-            # Map tool call id to index
-            id_to_index = {
-                block["id"]: i
-                for i, block in enumerate(content)
-                if block["type"] == "tool_use"
-            }
-            tool_calls = [{**tc, "index": id_to_index[tc["id"]]} for tc in _tool_calls]
```

Comment:
I checked `mypy` for errors, but could not find where "index" was referenced, so I removed the whole thing.